install_and_start_py.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+function install_libs_py {
+    echo "installing py libs"
+    if [ "$TESTDROID" == "1" ]; then
+        sudo easy_install pip
+        sudo pip install Appium-Python-Client unittest-xml-reporting
+        #sudo brew install ios-webkit-debug-proxy
+
+        export JUNIT_REPORT_STACK=1
+        export JUNIT_REPORT_PATH="TEST-all.xml"
+        JUNIT_REPORT_NAME="$DEVICE_NAME"
+        JUNIT_REPORT_NAME=${JUNIT_REPORT_NAME:="No Device Name Set"}
```

Comment:
I recall some system, (maybe testdroid or jenkins) has issues with junit.xml parsing if the root name contains special characters or whitespaces. 

If there are issues then try filtering out all non-alphanumerals/whitespaces.